client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -173,6 +179,25 @@ class SharingServiceDescription extends Component {
 				args: { service: this.props.service.label },
 				context: 'Sharing: Publicize',
 			} );
+		} else if ( 'refresh-failed' === this.props.status ) {
+			if ( this.props.expires ) {
+				description = this.props.translate(
+					'Your %(service)s connection has expired. This happens sometimes. Click the "Reconnect" button to fix it before %(expiryDate)s',
```

Comment:
"This happens sometimes" feels a bit strange, and might imply that our system is unreliable. How about

> Please reconnect to {service} before your connection expires on {date}."